lib/rules/prefer-wait-for.ts

```diff
@@ -119,6 +120,7 @@ export default createTestingLibraryRule<Options, MessageIds>({
           const [arg] = callExpressionNode.arguments;
           const fixers = [];
 
+          // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
I don't know which condition is the rule complaining here since `arg` can be emtpy.

Reply:
The same explanation from https://github.com/testing-library/eslint-plugin-testing-library/pull/385#discussion_r633902429 could explain this then.